### ISSUE TITLE
Finish Survey then use back button creates additional survey

### ISSUE THREAD

**Issue by mark-cunningham** [mark-cunningham/Impact#1]
I completed the survey as a learner, albeit there was only 2 stages - choose a priority and then offer a comment.  I selected finish survey, but there was no where to go (didn't offer me another survey) so I used the back button to take my back to the select priority screen.  Then I toggled backwards and forwards 5 times between select priority screen and the comments screen/Finish survey screen.  I noted that every time I toggled backwards and forwards  it created another survey response - ie, when I went back to Dashboard and View Evaluation, it said there had been 5 pupil surveys completed.

In summary, if you complete the survey and press the back button, and then return to the complete survey screen, it records that a new survey has been completed, even though it hadn't.
